app/models/employer_staff_role.rb

```diff
@@ -10,10 +10,10 @@ class EmployerStaffRole
 
   embedded_in :person
 
-  field :is_owner, type: Boolean, default: true
+  field :is_owner, type: Mongoid::Boolean, default: true
```

Comment:
this custom type was not needed on the upgrade on DC/ME, was there any reason ? I'm worried of the implications

Reply:
I believe it is okay, I have seen these changes in DC Rails 6 upgrade as well.

Reply:
updated the PR, to only use `Mongoid::Boolean` when necessary similar to DC.